crc/storage.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: storage.k8s.io/v1
 metadata:
   name: local-storage
 provisioner: kubernetes.io/no-provisioner
-volumeBindingMode: WaitForFirstConsumer
+volumeBindingMode: Immediate
```

Comment:
`Immediate` is fine for our single-node CRC environments, but there is talk of changing the `make crc_storage` to something non-CRC specific.  The initial `install_yamls` context actually did not even include/involve CRC, but over time we sort of injected CRC into this repo.  I believe the plan is to eventually extricate it and refactor `install_yamls` to work with "any" OCP cluster (with certain constraints, of course).

Anyhow, this change is probably fine for now.  Just wanted to document the aforementioned concern for potential reconsideration later on.

Reply:
Yeah, could we leave the volumeBindingMode alone for now and just merge the changes you have below for accessModes?

Reply:
ack, works for me, the issue I hit was a combination of `accessModes` and `volumeBindingMode`, but having the right access mode is the blocker

Reply:
Ah yeah, sorry, I didn't mean to imply only CRC is ever single-node.  SNO is definitely out there and in-use regularly.